lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -7,50 +7,17 @@ module ForemanThemeSatellite
   # For now, modifiers are not supported by the generator.
   # Make sure the regexp works both with Ruby and JavaScript.
   # Entries are processed in order and first wins.
+  # Occurrences of "proxy" should be treated as a bug as it is unclear whether the author refers to a HTTP(S) proxy or a Capsule
   FOREMAN_BRAND = {
-    /%{proxy}/               => '%{proxy}',
-    /%{foreman}/             => '%{foreman}',
-    /\b[Pp]roxy [Hh][Tt][Tt][Pp]\b(?!-)/    => 'Proxy HTTP', # Workaround for French translation
-    /\b[Pp]roxies [Hh][Tt][Tt][Pp]\b(?!-)/ => 'Proxies HTTP', # Workaround for French translation
```

Comment:
We still need those, otherwise in French `Proxies HTTP` will be branded as `Capsules HTTP` (which is wrong).

Reply:
Oh right, I forgot French does this...
While we're at it, what about "non-latin alphabet" languages? I noticed that in, for example, Russian, "Create Smart Proxy" is translated to "Создать **Смарт Прокси**", which will definitely not be branded correctly.
Are there any plans on covering these languages? Otherwise, I'd check it out myself.

Reply:
My assumption was (and I have actually outlined it in the redme) that we don't want to translate names.
Meaning that we should translate "Create Smart Proxy" to "Создать Smart Proxy". 
I know that for Hebrew for example we do leave Latin names for such things. I have to admit I am not sure if it is considered an OK practice in other languages.

Reply:
I very much agree with the idea of not translating names. I checked the translation files and found the following:
Note: for most of these I had to rely on DeepL, so no promises...

| Language | Translation of "Smart Proxy" | Translation into English |
| ------------- | ------------- |  ------------- |
| Catalan  | servidor intermediari intel·ligent  | smart proxy server |
| Czech  | none | none |
| German  | none | none |
| English (US)  | none | none |
| English (UK)  | none | none |
| Spanish | proxy inteligente | smart proxy |
| Italian | none | none |
| Japanese | none | none |
| Georgian |  ჭკვიანი პროქსი | smart proxy |
| Korean | 스마트 프록시 | smart proxy |
| Polish | smart proxy / inteligentnych proxy | smart proxy |
| Portuguese (BR) | smart proxy / proxy Inteligente | smart proxy |
| Russian | Смарт Прокси | smart proxy |
| Chinese (ZN) | 智能代机 | smart proxy |
| Chinese (TW) | 智慧型代理伺服器 | smart proxy server |

Reply:
Hmm.. Interesting. I see in Russian and Korean it's just a transliteration (It's the same text, just in Cyrillic/Korean letters) for other languages here it's actually a translation: (Smart -> Intelligent). And for Georgian it's a mix - "Proxy" is transliterated, but "Smart" is translated.
Specifically for our issue, Polish and Portuguese may become problematic, since they would be branded as "Capsule Inteligente" for Portuguese.

Reply:
> I very much agree with the idea of not translating names. I checked the translation files and found the following:

I would expect all of them to be in the glossary:

https://app.transifex.com/foreman/foreman_2013-08-14/translate/#en/foreman_2013-08-14?q=text%3Asmart

Yet when I check some languages I don't see them.